docs/public-networks/reference/evm-tool.md

```diff
@@ -433,130 +430,118 @@ Displays the version information.
 
 <!--tabs-->
 
 # Syntax
 
 ```bash
---json
+evmtool code-validate --file=<file>
 ```
 
-<!--/tabs-->
-
-Provides an operation-by-operation trace of the command in JSON.
-
-Set this option for EVM Lab Fuzzing. Whether or not `--json` is set, a summary JSON object is printed to standard output for each state test executed.
-
-`--trace` is an alias for `--json`.
-
-### `[no]memory`, `trace.[no]memory`
-
-<!--tabs-->
-
-# Syntax
+# Example
 
 ```bash
---[no]memory
+evmtool code-validate --file=eof.txt
 ```
 
 <!--/tabs-->
 
-Setting `--nomemory` disables tracing the memory output for each operation. Setting `--memory` enables it. Memory traces are disabled by default.
+Allows [Ethereum object formatted (EOF)](https://eips.ethereum.org/EIPS/eip-3540) code to be validated.
 
-For memory heavy scripts, disabling memory traces may reduce the volume of JSON output.
+You can specify a file containing one or more EOF containers or EVM bytecode using the `--file` option.
+Each line in the file is considered a separate program.
 
-`--trace.[no]memory` is an alias for `--[no]memory`.
+#### Use command arguments
 
-### Use command arguments
-
-If you use command arguments, you can list one or more state tests. All the state tests are evaluated in the order they are specified.
+If you use command arguments, each argument is considered a separate program.
+If a code segment includes spaces, it must be contained in quotes.
 
 <!--tabs-->
 
 # Docker example
 
 ```bash
-docker run --rm -v ${PWD}:/opt/referencetests hyperledger/besu-evmtool:develop --json state-test /opt/referencetests/GeneralStateTests/stExample/add11.json
+docker run --rm hyperledger/besu-evmtool:develop code-validate "0xef0001 010008 020002-0007-0002 030000 00 00000002-02010002 59-59-b00001-50-b1 03-b1" 0xef0002 0xef00010100040200010001030000000000000000
 ```
 
 # CLI example
 
 ```bash
-evm --json state-test stExample/add11.json
+evmtool code-validate "0xef0001 010008 020002-0007-0002 030000 00 00000002-02010002 59-59-b00001-50-b1 03-b1" 0xef0002 0xef00010100040200010001030000000000000000
 ```
 
 <!--/tabs-->
 
-### Use standard input
-
-If no reference tests are passed in using the command line, the EVM tool loads one complete JSON object from standard input and executes that state test.
-
-<!--tabs-->
-
-# Docker example
+#### Use standard input
 
-```bash
-docker run --rm -i hyperledger/besu-evmtool:develop --json state-test < stExample/add11.json
-```
-
-# CLI example
+If no reference tests are passed in using the command line, the EVM tool loads and validates code
+from standard input.
+Each line is considered a separate program.
+Comment lines and blanks are ignored.
 
-```bash
-evm --json state-test < stExample/add11.json
-```
+### `state-test`
 
-<!--/tabs-->
+Allows the [Ethereum state tests](https://github.com/ethereum/tests/tree/develop/GeneralStateTests)
+to be evaluated.
+The only applicable options are [`--json`](#json-trace) and [`--nomemory`](#nomemory-tracenomemory).
```

Comment:
These are the only options.  But they are ones used by other fuzzing tools.  Perhaps change it to "Notable options are"

I can enumerate all the other options, but the `--help` option will enumerate all the currently supported commands.  (example: `evmtool state-test --help`.  I don't think they need to be fully enumerated here.  The only options unique to `state-test` are the "-index" options (`--data-index`, `--value-index`, `--gas-index`, `--fork-index`) that are used to limit which tests are run to a specific data/value/gas/fork index.